perun-openapi/openapi.yml

```diff
@@ -12032,6 +12032,20 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/resourcesManager/getAssignedResourcesWithStatus:
+    get:
+      tags:
+        - ResourcesManager
+      operationId: getAssignedResourcesWithStatus
+      summary: List all resources with statuses associated with a member's group.
+      parameters:
+        - $ref: '#/components/parameters/groupId'
```

Comment:
`memberId `should be used as the parameter. For the summary I think the description in javadocs is more clear than the one here, I would prefer that instead. The same in rpc javadoc.

Reply:
Thank you, updated.